Mathlib/Analysis/LocallyConvex/Polar.lean

```diff
@@ -64,6 +64,10 @@ theorem polar_mem (s : Set E) (y : F) (hy : y ∈ B.polar s) : ∀ x ∈ s, ‖B
 theorem zero_mem_polar (s : Set E) : (0 : F) ∈ B.polar s := fun _ _ => by
   simp only [map_zero, norm_zero, zero_le_one]
 
+theorem polar_nonempty (s : Set E) : Nonempty (B.polar s) := by
+  use 0
+  exact zero_mem_polar B s
+
```

Comment:
You can use
```suggestion
theorem polar_nonempty (s : Set E) : Set.Nonempty (B.polar s) := by
  use 0
  exact zero_mem_polar B s

instance polar.instNonempty (s : Set E) : Nonempty (B.polar s) :=
  (polar_nonempty s).to_subtype
```
if you need an instance (same below). Note that the theorem uses `Set.Nonempty`, not `_root_.Nonempty`.
Thanks!
bors d+

Reply:
Thanks, I've changed both the `polar_nonempty` results to use `Set.Nonempty`. I'll keep the instances out of this PR and add them to #16316 instead so they can be considered there.